lib/4.06.1/unstable/bsb_native.ml

```diff
@@ -14249,14 +14249,14 @@ end = struct
 #1 "bsb_templates.ml"
 (* This file has been generated by ocp-ocamlres *)
 let root = OCamlRes.Res.([
-  Dir ("basic", [
-    Dir ("src", [
-      File ("demo.ml",
+  Dir  ("basic", [
+    Dir  ("src", [
+      File  ("demo.ml",
         "\n\
          \n\
          let () = Js.log \"Hello, BuckleScript\"")]) ;
-    Dir (".vscode", [
-      File ("tasks.json",
+    Dir  (".vscode", [
```

Comment:
No idea on what's causing this spaces... is it using some kind of formatter?

Reply:
Huh, so it seems like the spaces are included in the `jscomp/bsb/bsb_templates` and ever since these files have never been checked in then?

Reply:
I dropped the files from this PR, since it is not related